figures/apps.py

```diff
@@ -57,3 +57,6 @@ class FiguresConfig(AppConfig):
                 }
             },
         }
+
+    def ready(self):
+        import figures.signals  # noqa pylint: disable=unused-import
```

Comment:
Pylint is complaining about `unused-variable` instead of `unused-import`.

```suggestion
        import figures.signals  # noqa pylint: disable=unused-variable
```